### PR TITLE
Update explanation of propagation + add a sample

### PR DESCRIPTION
**Pull request by natechapin**

### CODE REVIEW

**Comment by yoavweiss** [comment]
@natechapin - can you join the WICG to appease the IPR bots?

File: PostTaskPropagation.md [hunk 1]
```diff
@@ -42,8 +42,10 @@ scheduler.postTask(task1, { signal }, signal);
 
 Rather than forcing developers to explicitly pass signals everywhere, we are
 proposing to expose the current `TaskSignal` so it can be inherited across async
-boundaries. This would work for both task callbacks (e.g. `postTask`) and for
-Promises. For example:
+boundaries. The current `TaskSignal` is propgated through `postTask` and
```
Comment (shaseley) [inline PostTaskPropagation.md]: sp: s/propgated/propagated

File: PostTaskPropagation.md [hunk 2]
```diff
@@ -42,8 +42,10 @@ scheduler.postTask(task1, { signal }, signal);
...
+through any Promise chains that begin inside of a `postTask` task, but will not
+propgate through other callbacks (e.g., event handlers, `setTimeout`, etc.).
```
Comment (shaseley) [inline PostTaskPropagation.md]: sp: s/propgate/propagate

[the remark this answers is not in the document]
Comment (shaseley) [reply PostTaskPropagation.md]: Also, it's probably worth saying what scheduler.currentTaskSignal returns in those cases

File: sample-code/tasksignal-propagation.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+<div id='content'></div>
+
+<script>
+  async function runExample() {
+
+    function returnCurrentTaskSignal() {
+      return scheduler.currentTaskSignal;
+    }
+
+    async function propagationEqualityAndPriorities() {
```
Comment (shaseley) [inline sample-code/tasksignal-propagation.html]: Should we show off propagating through something non-postTask (like fetch), or is that difficult to do in a file://

Comment (natechapin) [reply sample-code/tasksignal-propagation.html]: Sure, added

File: PostTaskPropagation.md (hunk 2 above)
Comment (natechapin) [reply PostTaskPropagation.md]: Done.

[the remark this answers is not in the document]
Comment (natechapin) [reply PostTaskPropagation.md]: Added async clarification, and mentioned default behavior when there is no current TaskSignal

**Comment by shaseley** [comment]
lgtm, thanks!

As a potential follow-up (for either of us): I wonder if adding a smaller/simpler hello-world-ish example would be good too, maybe combining setPriority() etc.? Or maybe a more realistic/toy end-to-end example would be useful? Anyways, something to think about.

**Pull request merged**